File: Backend-part/ClubberTv.Core/Entities/Match.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static ClubberTV.Core.Types.Types;

namespace ClubberTV.Core.Entities
{
    public partial class Match
    {
        public Guid Id { get; set; }
        public required string Title { get; set; }
        public required string Competition { get; set; }
        public DateTime Date { get; set; }
        public int Status { get; set; }
        public int Duration { get; set; }
        public ICollection<User> Users { get; set; } = new List<User>();
    }
}

File: Frontend-part/src/app/Interfaces/Match.ts
export interface Match {
  id: string;
  title: string;
  competition: string;
  date: Date;
  status: MatchStatus;
  duration: number;
}

export enum MatchStatus {
  Live,
  Replay
}

export interface AddedMatch {
  title: string;
  competition: string;
  date: Date;
  status: MatchStatus;
  duration: number;
}
